packages/server/test/resources.e2e-spec.ts

```diff
@@ -1,3 +1,30 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+import { AppModule } from './../src/app.module';
+import { JwtAuthGuard } from './../src/auth/jwt-auth.guard';
+import { RolesGuard } from './../src/auth/roles.guard';
+
 describe('ResourcesController (e2e)', () => {
+    let app: INestApplication;
+
+  beforeEach(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    })
+    .overrideGuard(JwtAuthGuard)
+    .useValue({ canActivate: () => true })
+    .overrideGuard(RolesGuard)
+    .useValue({ canActivate: () => true })
+    .compile();
+
+    app = moduleFixture.createNestApplication();
+    await app.init();
+  });
 
+  it('/resources (GET)', () => {
+    return request(app.getHttpServer())
+      .get('/resources')
+      .expect(200);
```

Comment:
Would be nice to check payload as well :)
Add TODO if you don't want to do it now.

Reply:
Got it. I'll add comparing with the mock data